doc_src/index.hdr.in

```diff
@@ -17,7 +17,199 @@
+
+- <b>Extensive UI</b>: *fish* supports the user with syntax highlighting, autosuggestions, tab completions and selections lists, that can be navigated and filtered, see: <a href="#autosuggestions">Autosuggestions</a> and <a href="#completion">Tab Completions</a>.
+
+- <b>No further configuration is needed</b>: *fish* comes preconfigured so that it will be an efficient helper on the commandline out of the box.
+
+- <b>Add new commands easily</b>: in *fish* new commands can be added on the fly. The syntax is easy to learn and there is no administrative overhead, see: <a href="#syntax-function">Functions</a>.
+
+- <b>Interactive shell</b>: *fish* focuses on commands that will be run in an interactive session: While it supports job control for external commands, its own commands share the process of the shell, that started them.
+
+<a href="#introduction"><-</a><hr>
+
+
+\section shell What is a Shell
+
+This section is about the basics of shells. *fish* is a shell and shells have a lot in common:
+
+- <a href="#shell-standards">Shell Standards</a>: Why shells adjust to standards
+- <a href="#man-page">Manual Pages</a>: Commands usually come with a standardized manual page
+- <a href="#shell-syntax">Command Syntax</a>: Shell commands have a standard syntax
+- <a href="#shell-programming">Commands versus Programs</a>: Commands differ from normal programs
+- <a href="#shebang-line">Shebang Line</a>: How the shell knows the language of a script
+
+\subsection shell-standards Shell Standards
+
+A shell is an interface to the operating system that reads from the commandline of a terminal. A shell's task is to identify and interpret commands. The commands can come from different applications and can be written in different programming languages.
+
+This can only work smoothly if shells adapt to some common standards. For shells there is the POSIX standard, see <a href="https://en.wikipedia.org/wiki/Command-line_interface#Command-line_interpreter">Command-line interpreters</a>. `fish` tries to satisfy the POSIX standard wherever it does not get into the way of its own design principles, see <a href="design.html">`fish` Design</a>.
+
+\subsection man-page Manual Pages
+
+There is a common standard on how to receive help on shell commands: applications provide a manual page to their commands that can be opened with the `man` command:
+
+\fish
+>man COMMAND
+\endfish
+
+This convention helps to make sure help can be found on commands no matter where they originate from. *fish*'s internal commands all come with a manual page.
+
+\subsection shell-syntax Command Syntax
+
+Shells also support some common syntax for executing commands. That way a command can be started in the same way, regardless of the application, where it comes from, and the shell, where it is executed in. 
```

Comment:
I think introducing shells as a way of starting commands is a great idea, but I wouldn't talk about other shells here. Rather than how to use all shells, I think this section should focus on how to use fish as a shell.

Perhaps it would be better to drop this and instead make the "syntax overview" section more approachable?

Reply:
Hmm, I wil try my best to rewrite this section, see above. 

I just feel this is one point you have to understand about shells in order to understand fish: they are not build on an empty field, they have to measure up to certain requirements in order to serve the purpose they are intended for. 

Ususally you build program and then they know all about themselves or other applications they are meant to deal with. But a shell has to deal with unknown future applications. That is very special. And I struggled initially in comprehending how this is done. I felt this to be an important point in explaining what a shell is all about. 

As a beginner I am told that fish is a shell. So my next question would be what is a shell. Where is fish just behaving like any other CLI shell and where is it a fish feature that is just about fish and not about fish being a shell. 

The status is a good example. fish calls the status $status. That is special, but other then that any CLI shell must deal with the exit code of commands in some way and this exit code and its values is not special to fish, but part of the operating system. Do you understand what I mean?

Do you want me to skip that whole section on "What is a shell?"? I will give it a try just now and try to rewrite it above consindering your comments in my rewrite.